app/src/main/java/com/mstudio/superheroarch/Character.kt

```diff
@@ -1,15 +1,34 @@
 package com.mstudio.superheroarch
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 import java.io.Serializable
 
+@Entity(tableName = "characters")
```

Comment:
you don't need to put the tableName, just @Entity is enough

Reply:
Fixed in next comit